src/chains/index.ts

```diff
@@ -108,7 +108,6 @@ export { filecoinCalibration } from './definitions/filecoinCalibration.js'
 export { filecoinHyperspace } from './definitions/filecoinHyperspace.js'
 export { flare } from './definitions/flare.js'
 export { flareTestnet } from './definitions/flareTestnet.js'
```

Comment:
This will be a breaking change, let's add a `@deprecated` flag meaning it will be removed in the next major version

```suggestion
export { flareTestnet } from './definitions/flareTestnet.js'
/** @deprecated Use 
export { flowPreviewnet } from './definitions/flowPreviewnet.js'
```